### PR TITLE
SCYLLA-VERSION-GEN: remove unnecessary bashism

### PR DESCRIPTION
**Pull request by tchaikov**
remove unnecessary bashism, so that this script can be interpreted by a POSIX shell.

/bin/sh is specified in the shebang line. on debian derivatives, /bin/sh is dash, which is POSIX compliant. but this script is written in the bash dialect.

before this change, we could run into following build failure when building the tree on Debian:

[7/904] ./SCYLLA-VERSION-GEN
./SCYLLA-VERSION-GEN: 38: [[: not found

after this change, the build is able to proceed.

closes #25
Signed-off-by: Kefu Chai <kefu.chai@scylladb.com>

### CODE REVIEW

File: SCYLLA-VERSION-GEN [hunk 1]
```diff
@@ -35,7 +35,7 @@ usage() {
 }
 
 OVERRIDE=
-while [[ $# > 0 ]]; do
+while [ $# > 0 ]; do
```
Comment (nyh) [inline SCYLLA-VERSION-GEN:38]: This change is wrong. Did you test it?
"[" is not special shell syntax, it is just /bin/[, so the ">" does what it does in a shell - redirection.
You were looking for "-gt", not ">".
This patch was already merged so please post a follow-up patch.
CC @tchaikov @avikivity

Comment (tchaikov) [reply SCYLLA-VERSION-GEN:38]: @nyh just saw your message. 

TL;DR

i failed to test it properly. 

long story

i have two local repos for 
`scylla-python3`, one in scylladb as a submodule, the other is sitting in my `$HOME/dev`. so what i tested was the submodule where i changed the shebang to bash. but when i was trying to create a pull request based on my change, i found that #25 was a better fix. so i just copied from it without revisiting it carefully.

will test the change properly next time.
